scripts/release/orthoinference/.gitignore

```diff
@@ -1,2 +1,4 @@
 [0-9]*
 data-release-pipeline
+*.out
+runOrthoinference.sh
```

Comment:
Why ignore this script? Isn't it a part of the release?

Reply:
Yes, but this is just ignoring the symbolic link at the 'root' orthoinference directory created to the actual script inside the data-release-pipeline repository.